packages/contracts/contracts/GenArt721CoreV3.sol

```diff
@@ -1061,6 +1062,28 @@ contract GenArt721CoreV3 is
         emit ProjectUpdated(_projectId, FIELD_PROJECT_SCRIPT);
     }
 
+    /**
+     * @notice Adds a pre-compressed script to project `_projectId`.
```

Comment:
I think it is worth adding some more details to this function's notice (+ updateProjectScriptCompressed) about what uploading the compressed script is doing. I think some important details to include are:
* a string can be pre-compressed (currently in an @dev comment, I think should just be included in @notice)
* the compressed string is stored in a compressed form on-chain
* no change to read behavior: during reads, the compressed representation is ran through an on-chain decompression algorithm, and the uncompressed string is returned without reliance on any external dependencies

Reply:
Good call! Updated the notice tag to include these notes 🙏